### ISSUE TITLE
[Documentation] Missing example in GraphConv tutorial

### ISSUE THREAD

**Issue by nashutosh** [deepchem/deepchem#4500]
### Problem
The `GraphConvModel` section in tutorials (http://deepchem.io/docs) explains theory but lacks a minimal runnable example. New users may struggle to instantiate the model or load data.

### Steps to Reproduce
1. Visit the documentation section for `GraphConvModel`  
2. Notice there is no code snippet demonstrating dataset loading, model training, or evaluation

### Expected Behavior
Include a code snippet like:
```python
from deepchem.models import GraphConvModel
from deepchem.data import NumpyDataset

# Example data
X, y = load_some_graph_data()
model = GraphConvModel(...)
model.fit(dataset)
print(model.evaluate(dataset, ["accuracy"]))

**Comment by dakshhhhh16** [comment]
Hi!  I'd like to work on this issue.  #4500

**Comment by Divansu47** [comment]
Subject: GSSoC '26: Ready to add runnable example for GraphConvModel



Hi @ashutosh-engineer and maintainers,

I am participating in GSSoC '26 and would love to contribute to DeepChem by resolving this issue.

I have reviewed the GraphConvModel documentation and confirmed that a runnable example is missing. I have already drafted a solution using the Delaney (Solubility) dataset, which is standard for quick demonstrations.

My Proposed Plan:

Locate the relevant documentation source file (likely in docs/source/ or the tutorial notebooks).

Insert a clear, runnable code block that demonstrates:

Data loading with the correct GraphConv featurizer.

Model instantiation.

Training (fit).

Evaluation (evaluate).

Verify the snippet runs without errors in a local environment.

Submit a PR for review.

Proposed Code Snippet: Here is the working example I intend to add. I have tested this locally, and it successfully trains and evaluates the model:
```
PYTHON 
import deepchem as dc
from deepchem.models import GraphConvModel
import numpy as np

# 1. Load Data (Delaney - Solubility)
# It is crucial to use 'GraphConv' featurizer for GraphConvModel
tasks, datasets, transformers = dc.molnet.load_delaney(featurizer='GraphConv')
train_dataset, valid_dataset, test_dataset = datasets

# 2. Initialize Model
# We use 'regression' mode as we are predicting solubility values
model = GraphConvModel(n_tasks=len(tasks), mode='regression', batch_size=50, dropout=0.2)

# 3. Train the Model
print("Training model...")
model.fit(train_dataset, nb_epoch=10)

# 4. Evaluate
print("Evaluating model...")
metric = dc.metrics.Metric(dc.metrics.pearson_r2_score)
train_score = model.evaluate(train_dataset, [metric], transformers)
test_score = model.evaluate(test_dataset, [metric], transformers)

print(f"Training set score: {train_score}")
print(f"Test set score: {test_score}")
```
I am ready to proceed immediately. Could you please assign this issue to me?

Thanks!
